reverse.js

```diff
@@ -157,9 +157,16 @@
       }
       else if(lop === "t") {
         for (a = 0; a < alen; a += 2) {
-          // reflect previous cx/cy over x/y
-          cx = x + (x-cx);
-          cy = y + (y-cy);
+          if (["t", "q"].indexOf(prevop) > -1) {
+            // reflect previous cx/cy over x/y
+            cx = x + (x-cx);
+            cy = y + (y-cy);
+          }
+          else {
+            cx = x;
+            cy = y;
+            prevop = lop;
+          }
```

Comment:
Let's use the same code as for s/c though (universal x/y assignment with conditional increment)

Reply:
Doesn't work in that case. With s/c we reflect cx2/cy2 here we have cx/cy. If we assign them first it breaks

Reply:
d'uh, yes of course.